EncryptedDATAStack.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'EncryptedDATAStack'
-  s.version          = '7.0.3'
+  s.version          = '8.0.0'
```

Comment:
The version number is meant to be in sync with [DataStack](https://github.com/3lvis/DATAStack). I guess it could be changed to 7.0.3.1 to avoid any overlap.

Reply:
Entirely up to yourself :) @flipacholas - Thanks for taking a look I will leave the pull request for you to either close or accept after amending to suit your needs.